src/poetry/inspection/lazy_wheel.py

```diff
@@ -85,6 +89,14 @@ def metadata_from_wheel_url(
         # themselves are invalid, not because we've messed up our bookkeeping
         # and produced an invalid file.
         raise InvalidWheel(url, name)
+    except Exception as e:
+        if isinstance(e, LazyWheelUnsupportedError):
```

Comment:
We probably should at least add a debug log with the error message.

Reply:
It will pop up in the stack trace anyway. But yeah adding debug log cannot hurt.